src/routes/posts.ts

```diff
@@ -0,0 +1,34 @@
+import express from "express";
+import Post from "../handlers/post-handler";
+
+const postMethods = new Post;
+
+export const posts = {
+  get: (req: express.Request, res: express.Response) => {
+    postMethods.showPosts(req, res);
```

Comment:
**bad solution**

Method `showPosts` (or `getPosts`) should return a list of posts for u. Single responsibility! 

after that, you will put this data to response